examples/todo-pylist.html

```diff
@@ -83,7 +77,7 @@ <h1>To Do List</h1>
                     </button>
                 </div>
 
-                <py-list id="myList"></py-list>
+                <py-list src="./pylist.py" id="myList"></py-list>
```

Comment:
I don't understand what's going on here... what's going on here?

Reply:
I agree, In fact, I wanted to chat about this as well.